infrastructure/cover-service-importers/templates/configmap.yaml

```diff
@@ -31,6 +31,7 @@ data:
   APP_SUPERVISOR_QUEUES: {{ join " " .Values.app.supervisor.queues | quote }}
   APP_NO_HITS_TTL: {{ mul .Values.app.no_hits_ttl 1000 | quote }}
   APP_NO_HITS_CACHE_SINGLE_COVER_TTL: {{ mul .Values.app.no_hits_single_cover_ttl 1000 | quote }}
+  APP_FORSIDERDK_SUBFOLDERS: {{ .Values.app.vendor.forsiderdk_subfolders | quote }}
```

Comment:
Unsure about the format... `| qoute` her will "" the whole thing as an string... Not sure about helm and arrays

Reply:
I want the whole thing as a string. Symfony handles string -> array